collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java

```diff
@@ -0,0 +1,111 @@
+package org.odk.collect.android.regression;
+
+import androidx.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.odk.collect.android.support.pages.AboutPage;
+import org.odk.collect.android.support.pages.MainMenuPage;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.odk.collect.android.support.matchers.DrawableMatcher.withImageDrawable;
+import static org.odk.collect.android.support.matchers.RecyclerViewMatcher.withRecyclerView;
+
+//Issue NODK-234
+@RunWith(AndroidJUnit4.class)
+public class AboutPageTest extends BaseRegressionTest {
+
+    @Test
+    public void when_rotateScreenOnAboutPage_should_notCrash() {
+        //TestCase1
+        new MainMenuPage(rule)
+                .clickOnMenu()
+                .clickAbout()
+                .rotateToLandscape(new AboutPage(rule))
+                .assertOnPage()
+                .scrollToOpenSourceLibrariesLicenses();
+    }
+
+    @Test
+    public void when_openAboutPage_should_iconsBeVisible() {
```

Comment:
I'm not sure we need this as an automated test. In my opinion, it's unlikely that the icons would dissapear and if they did it wouldn't a huge problem. Do you think we could just delete this test?

Reply:
I think that automation of all cases that are performed manually in the regression cycle is valuable and can shorten the time of performing the whole cycle. It is something that we are checking anyway so why not check that automatically instead?

Reply:
Right that's a good point. I think we probably need to start thinking about seperating out `regression` from the other Espresso tests so we can have one faster set to run locally as devs but keep the value of having all these regression test cases automated - potentially running then before PR merge or once a day or something.

Reply:
> I think we probably need to start thinking about seperating out regression from the other Espresso tests so we can have one faster set to run locally

I think this is a really good idea. Is introducing a new Gradle task that runs the non-regression tests a reasonable way to do this?

Reply:
@lognaturel I think there is a way to filter on annotations but not sure about packages. Will have a look into this.